src/main/java/org/graylog/integrations/aws/transports/KinesisConsumer.java

```diff
@@ -9,6 +9,7 @@
 import org.graylog2.plugin.system.NodeId;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
```

Comment:
delete unused imports `org.graylog.integrations.aws.service.AWSService;` 
and `software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;`